PubHub/PubHub.API/Domain/Seeding/AccountSeed.cs

```diff
@@ -43,7 +43,8 @@ public override void Configure(EntityTypeBuilder<Account> builder)
                     EmailConfirmed = true,
                     LastSignIn = DateTime.UtcNow,
                     SecurityStamp = "UserSeedSecurityStamp",
-                    PhoneNumber = "4587654321"
+                    PhoneNumber = "4587654321",
```

Comment:
Why are we adding phonenumbers?

Reply:
They've always been there. I just added a comma to enter a value for another property below 🙂

Reply:
Not really 😅